ckanext/unaids/helpers.py

```diff
@@ -139,3 +140,9 @@ def get_extstorage_resource_filename(resource):
         url_path = urlparse(resource['url']).path
         return path.basename(url_path)
     return resource['url']
+
+
+def get_max_resource_size():
```

Comment:
It's better to use the build in core ckan method `ckan.lib.uploader.get_max_resource_size`